<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P2M</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>To:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="lnTo">
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>SW Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lnName"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Version:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lnVersion"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>SW Path:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lnPath"/>
        </item>
        <item>
         <widget class="QToolButton" name="btSetPath">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Changelog:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtChangelog"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnSend">
      <property name="text">
       <string>&amp;Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lnTo</tabstop>
  <tabstop>lnName</tabstop>
  <tabstop>lnVersion</tabstop>
  <tabstop>lnPath</tabstop>
  <tabstop>txtChangelog</tabstop>
  <tabstop>btnSend</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
